src/components/secure-backup/index.tsx

```diff
@@ -61,92 +76,102 @@ export class SecureBackup extends React.PureComponent<Properties, State> {
     return this.props.backupExists && !this.props.isBackupRecovered;
   }
 
-  renderBackedUpMode() {
-    const { isLegacy } = this.props;
-    const button = isLegacy ? <Button onPress={() => this.props.onGenerate()}>Generate new backup</Button> : null;
+  get isSystemPrompt() {
+    return this.backupStage === BackupStage.SystemPrompt;
+  }
+
+  renderHeader = () => {
     return (
-      <div {...cn('backed-up')}>
-        <p>This session is backing up your keys.</p>
-        <p>This backup is trusted because it has been restored on this session.</p>
-        {this.renderButtons(button)}
+      <div {...cn('header')}>
+        <h3 {...cn('title')}>Account Backup</h3>
+        <IconButton {...cn('close')} Icon={IconXClose} onClick={this.props.onClose} size={40} />
       </div>
     );
-  }
+  };
 
-  renderNoBackupMode() {
+  renderVideoBanner = () => {
     return (
-      <>
-        <p>
-          Your keys are <b>not being backed up from this session.</b>
-        </p>
-        <p>Back up your keys before signing out to avoid losing them.</p>
-        {this.renderButtons(<Button onPress={() => this.props.onGenerate()}>Generate backup</Button>)}
-      </>
+      <div {...cn('video-banner')}>
+        <video
+          {...cn('video')}
+          // TODO: uncomment when env vars updated
+          // src={`${this.props.assetsPath}/E2EEncryption.mp4`}
```

Comment:
I will also remove this comment and update once I merge the env var PRs that I have opened re update asset path variables/config